Makefile

```diff
@@ -226,6 +232,10 @@ test\:lens:
 	@$(MAKE) deps:lens
 	gotestsum --format testname -- ./$(LENS_TEST_DIRECTORY)/... $(TEST_FLAGS)
 
+.PHONY: test\:cli
+test\:cli:
+	gotestsum --format testname -- ./$(CLI_TEST_DIRECTORY)/... $(TEST_FLAGS)
```

Comment:
question: For `lens` and `cli` I noticed we are also using `testname` over `pkgname`? Is that desired? I find `testname` to be useful in ci, but locally I prefer `pgkname`

Reply:
I just copy-pasted the settings from the existing commands we had.  I actually don't really care for either of them and other than for a quick sanity check that it has actually run the tests, I would probably be happy with just the count and any failure logs.

Reply:
Because it is being used in the ci command can leave as is